### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by ibrahemomari**

### CODE REVIEW

File: control/content/app.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
                     controllerAs: 'ContentChat',
                     controller: 'ContentChatCtrl'
                 })
-                .when('/chat/:userToken', {
+                .when('/chat/:encodedReview', {
```
Comment (mas-iota) [inline control/content/app.js:18]: - Use the framework's state to store the reviews collection.
- Then access the target review by sending its userId as a query parameter.
- Manage the framework's state (update/delete/create)

Comment (mas-iota) [reply control/content/app.js:18]: `/chat/:userId`

File: control/content/app.services.js [hunk 2]
```diff
@@ -14,6 +14,10 @@
         get: function (_tagName) {
           var deferred = $q.defer();
           Buildfire.datastore.get(_tagName, function (err, result) {
+            if (!result || !result.data || !result.data.introduction || Object.keys(result.data).length === 0) {
+              result.introduction = '';
```
Comment (mas-iota) [inline control/content/app.services.js:18]: replace `result.introduction = '';` to `result.data.introduction = '';`

File: control/content/assets/style.css [hunk 3]
```diff
@@ -1,17 +1,41 @@
...
+.feedback-calculations{
+    border-radius: 4px;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
 }
-.review-item:nth-child(even){
-    background:#eef0f0;
+
+.section__title {
```
Comment (mas-iota) [inline control/content/assets/style.css:8]: change to `section-title`

File: widget/controllers/widget.home.controller.js [hunk 4]
```diff
@@ -81,8 +94,19 @@
               if (!$scope.$$phase)
                   $scope.$digest();
           };
+        
+        const getStrings = ()=>{
+            stringsKeys.forEach((key)=>{
+                buildfire.language.get({stringKey: key}, (err, result) => {
+                if (err) return console.error("Error while retrieving string value", err);
+                WidgetHome[key.split('.')[1]] = result;
```
Comment (mas-iota) [inline widget/controllers/widget.home.controller.js:102]: add values to WidgetHome.translations or think of a better approach

File: widget/controllers/widget.home.controller.js [hunk 5]
```diff
@@ -81,8 +94,19 @@
...
+                });
+                
+            });
+        }
 
         function init() {
+            getStrings();
```
Comment (mas-iota) [inline widget/controllers/widget.home.controller.js:109]: Make sure getStrings is not falling in racing conditions

**Review by mas-iota — CHANGES_REQUESTED** [review]

**Review by charkhaw — APPROVED** [review]

**Pull request merged**